.travis.yml

```diff
@@ -13,12 +15,19 @@ before_install:
   - echo "$TRAVIS_COMMIT_MSG" | grep '\[skip sauce\]'; export TWBS_DO_SAUCE=$?; true
   - if [ "$TRAVIS_REPO_SLUG" = twbs-savage/bootstrap ]; then export TWBS_DO_VALIDATOR=0; fi
 install:
+  - bundle install --deployment --jobs=3
+  - gem install scss_lint --no-rdoc --no-ri
```

Comment:
there's one in root which bundle install is using, but --deployment vendors them locally for caching & whichever grunt scss lint plugin we use doesn't support bundle exec